src/material-examples/text-field-autosize-basic/text-field-autosize-basic-example.css

```diff
@@ -0,0 +1,15 @@
+div {
+  margin-right: 12px;
+}
+
+label {
+  color: rgba(0, 0, 0, 0.54);
+  display: block;
+  margin: 0.8em 0 0.4em;
+}
+
+textarea {
+  width: 100%;
+  margin-bottom: 1.25em;
+}
+
```

Comment:
These demo styles are actually a bit too broad. Can you add CSS classes (prefixed with `demo-`) and target those instead?